src/components/views/right_panel/RoomHeaderButtons.tsx

```diff
@@ -21,82 +21,82 @@ limitations under the License.
 import React from 'react';
 import { _t } from '../../../languageHandler';
 import HeaderButton from './HeaderButton';
-import HeaderButtons, {HEADER_KIND_ROOM} from './HeaderButtons';
-import {RIGHT_PANEL_PHASES} from "../../../stores/RightPanelStorePhases";
+import HeaderButtons, {HeaderKind} from './HeaderButtons';
+import {RightPanelPhases} from "../../../stores/RightPanelStorePhases";
 import {Action} from "../../../dispatcher/actions";
 import {ActionPayload} from "../../../dispatcher/payloads";
 
 const MEMBER_PHASES = [
-    RIGHT_PANEL_PHASES.RoomMemberList,
-    RIGHT_PANEL_PHASES.RoomMemberInfo,
-    RIGHT_PANEL_PHASES.EncryptionPanel,
-    RIGHT_PANEL_PHASES.Room3pidMemberInfo,
+    RightPanelPhases.RoomMemberList,
+    RightPanelPhases.RoomMemberInfo,
+    RightPanelPhases.EncryptionPanel,
+    RightPanelPhases.Room3pidMemberInfo,
 ];
 
 export default class RoomHeaderButtons extends HeaderButtons {
     constructor(props) {
-        super(props, HEADER_KIND_ROOM);
-        this._onMembersClicked = this._onMembersClicked.bind(this);
-        this._onFilesClicked = this._onFilesClicked.bind(this);
-        this._onNotificationsClicked = this._onNotificationsClicked.bind(this);
+        super(props, HeaderKind.Room);
+        this.onMembersClicked = this.onMembersClicked.bind(this);
+        this.onFilesClicked = this.onFilesClicked.bind(this);
+        this.onNotificationsClicked = this.onNotificationsClicked.bind(this);
     }
 
     onAction(payload: ActionPayload) {
```

Comment:
private

Reply:
This should be public since the subclass uses it.

Reply:
if a subclass is using it then shouldn't it be `protected`?

Reply:
Oops should have paid more attention in those OOP classes.